includes/class-sensei-lesson.php

```diff
@@ -1118,7 +1118,11 @@ private function save_post_meta( $post_key = '', $post_id = 0 ) {
 
 		// Check if the user has permission to edit the target course.
 		if ( 'lesson_course' === $post_key && ! current_user_can( get_post_type_object( 'course' )->cap->edit_post, $new_meta_value ) ) {
-			return;
+			if ( empty( $new_meta_value ) ) {
+				$new_meta_value = '';
+			} else {
+				return;
```

Comment:
As we are updating this method, can we remove the `@return int|bool` from the docs?